src/util/webhooks.ts

```diff
@@ -0,0 +1,30 @@
+import { getLogger } from "../config/logger";
+import statsd from "../config/statsd";
+import { metricWebhooks } from "../config/metric-names";
+
+export const getCurrentTime = () => new Date();
+
+export const calculateProcessingTimeInSeconds = (
+	webhookReceivedTime: Date,
+	webhookName: string,
+	status?: number,
+): number => {
+	const logger = getLogger("webhookProccessingTime");
```

Comment:
We need to use logger from the context from where it called. Otherwise we are loosing context parameters
- [ ] Pass context logger and use it here (req.log or context.log)